src/ui_contextmenu.cpp

```diff
@@ -31,6 +32,9 @@ class OpenLyricsContextSubItem : public contextmenu_item_simple
             case cmd_manualsearch_lyrics: out = "Search for lyrics (manually)"; break;
             case cmd_edit_lyrics: out = "Edit lyrics"; break;
             case cmd_mark_instrumental: out = "Mark as instrumental"; break;
+            case cmd_seek_to_next_lyric: out = "Seek to next lyric timestamp"; break;
+            case cmd_seek_to_prev_lyric: out = "Seek to prev lyric timestamp"; break;
+            case cmd_seek_to_repeat_current_lyric: out = "Seek to repeat current lyric timestamp"; break;

```

Comment:
I think a better name would be more to the effect of "Seek to the current lyric timestamp" because that's what it's doing. The repeat is a 2nd-order effect that only applies if it's currently playing.